main.js

```diff
@@ -193,15 +192,19 @@ function showMyLog() {
 function checkLocalStorage() {
   if (!JSON.parse(localStorage.getItem('activityCards'))) {
     activityCards = [];
+    showElement(emptyLog1);
+    showElement(emptyLog2);
   } else {
     activityCards = JSON.parse(localStorage.getItem('activityCards'));
+    hideElement(emptyLog1);
+    hideElement(emptyLog2);
     showMyLog();
```

Comment:
Ensures that the 'you haven't logged any activities yet' message does not display if there are activity cards in the log.